code/game/objects/items/rogueweapons/melee/swords.dm

```diff
@@ -615,3 +615,38 @@
 	smelt_bar_num = 2
 	max_blade_int = 150
 	max_integrity = 200
+
+/obj/item/rogueweapon/sword/long/blackflamb
+	force = 20
+	force_wielded = 32
+	possible_item_intents = list(/datum/intent/sword/cut, /datum/intent/sword/thrust, /datum/intent/sword/strike)
+	gripped_intents = list(/datum/intent/sword/cut, /datum/intent/sword/thrust, /datum/intent/sword/strike, /datum/intent/sword/chop)
+	icon_state = "blackflamb"
+	/obj/item/rogueweapon/sword/long/blackflamb
+	force = 20
+	force_wielded = 32
+	icon_state = "blackflamb"
+	name = "blacksteel flamberge"
+	desc = "A strange sword with a winding blade forged of blacksteel and a rontz pommel."
+	smeltresult = /obj/item/ingot/blacksteel
+	max_integrity = 200
+	lefthand_file = 'icons/mob/inhands/weapons/roguebig_lefthand.dmi'
+	righthand_file = 'icons/mob/inhands/weapons/roguebig_righthand.dmi'
+	name = "blacksteel flamberge"
+	desc = "A strange sword with a winding blade forged of blacksteel and a rontz pommel."
+	parrysound = "bladedmedium"
+	swingsound = BLADEWOOSH_LARGE
+	pickup_sound = 'sound/foley/equip/swordlarge2.ogg'
+	bigboy = 1
+	wlength = WLENGTH_LONG
+	gripsprite = TRUE
+	pixel_y = -16
+	pixel_x = -16
+	inhand_x_dimension = 64
+	inhand_y_dimension = 64
+	associated_skill = /datum/skill/combat/swords
+	throwforce = 15
+	thrown_bclass = BCLASS_CUT
+	dropshrink = 0.75
+	smeltresult = /obj/item/ingot/blacksteel
+	max_integrity = 200
```

Comment:
sorry lol, i didn't think about the way github would insert this. try:

```suggestion
	/obj/item/rogueweapon/sword/long/blackflamb
	force = 20
	force_wielded = 32
	icon_state = "blackflamb"
	name = "blacksteel flamberge"
	desc = "A strange sword with a winding blade forged of blacksteel and a rontz pommel."
	smeltresult = /obj/item/ingot/blacksteel
	max_integrity = 200
```

Reply:
To be fair I should have checked and fixed this myself, but I appreciate you coming back to fix it